### DISCUSSION TITLE
A32nx LVAR for Dome light

### DISCUSSION THREAD

**Discussion by Peter-Mary-Jo** [flybywiresim/aircraft#6102] [Help] [answered]
Hello Dev community,
Is there a LVAR to command the Dome Light? I don't find any in the LVAR list but it has perhaps a specific name.
Many thanks for your works

**Comment by BlueberryKing** [comment, marked as answer]
Hello!
Looking at the code, it looks like we use the `A:LIGHT CABIN:1` and `A:LIGHT POTENTIOMETER:7` aircraft variables for the dome light. `A:LIGHT CABIN:1` uses `0`, `0.5` and `1` for the positions `OFF`, `DIM` and `BRT` respectively, whereas `A:LIGHT POTENTIOMETER:7` uses `0`, `50` and `100`.

The variables are set via the `K:2:CABIN_LIGHTS_SET` and `K:LIGHT_POTENTIOMETER_7_SET` events.

I hope this helps. Let us know if there's anything else you need to know. 😄

**Comment by Peter-Mary-Jo** [comment]
Many thanks for this detailed answer to my question. I'll be able to bind a button to light up the flight deck from C&D
